src/lib/hash/blake2/blake2b.h

```diff
@@ -10,13 +10,16 @@
 
 #include <botan/hash.h>
 #include <botan/sym_algo.h>
+#include <botan/internal/alignment_buffer.h>
 #include <memory>
 #include <string>
 
 namespace Botan {
 
 class BLAKE2bMAC;
 
+enum blake2b_constant { BLAKE2B_BLOCKBYTES = 128, BLAKE2B_IVU64COUNT = 8 };
```

Comment:
There doesn't seem to be a reason to expose `BLAKE2B_IVU64COUNT` here

Reply:
True. I just used `std::array<>` in the *.cpp file and now use `blakeIV.size()` where needed.